File: aisile-manager-web/src/main/webapp/js/service/brandService.js
//封装所有连接后端代码    需要$http
app.service('brandService',function($http){
	//查询全部的数据
	this.findAll = function(){
		return $http.get('../brand/findAll.do');
	}
	//分页
	this.findPage = function(page,rows){
		return $http.get('../brand/findPage.do?page='+page+"&rows="+rows);
	}
	//分页带数据查询
	this.findSearch = function(page,rows,searchEntity){
		return $http.post('../brand/findSearch.do?page='+page+"&rows="+rows,searchEntity);
	}
	//修改
	this.update = function(entity){
		return $http.post('../brand/update.do',entity);
	}
	//添加
	this.add = function(entity){
		return $http.post('../brand/add.do',entity);
	}
	//根据id查询对象
	this.findOne = function(id){
		return $http.post('../brand/findOne.do?id='+id);
	}
	//删除
	this.dele = function(selectIds){
		return $http.get('../brand/dele.do?ids='+selectIds);
	}
	//下拉框
	this.selectOptionList = function(){
		return $http.get('../brand/selectOptionList.do');
	}
})

File: aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/BrandService.java
package com.aisile.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.aisile.pojo.TbBrand;
import com.aisile.pojo.entity.PageResult;

public interface BrandService {
	/**
	 * 查询全部数据
	 * @return
	 */
	public List<TbBrand> geTbBrands();
	/**
	 * 分页
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 模糊查询
	 * @param pageNum
	 * @param pageSize
	 * @param tbBrand
	 * @return
	 */
	public PageResult findSearch(int pageNum,int pageSize,TbBrand tbBrand);
	/**
	 * 添加方法
	 * @param tnBrand
	 */
	public void add(TbBrand tnBrand);
	/**
	 * 修改方法
	 * @param tbBrand
	 */
	public void update(TbBrand tbBrand);
	/**
	 * 回显   返回的是一个对象
	 * @param id
	 * @return
	 */
	public TbBrand findOne(Long id);
	/**
	 * 删除(批量删除)
	 * @param ids
	 */
	public void delete(Long [] ids);
	/**
	 * 查询下拉框
	 * @return
	 */
	public List<Map> selectOptionList();
}
